README.md

```diff
@@ -77,6 +77,171 @@ There is also support for using the dynamic gateway client by relying on the pee
 
 Support for server-based gateway support, available in Fabric 2.4, is coming soon.
 
+### Structured Data Support for Transaction Input with Schema Validation
+When calling the `POST /transactions` endpoint, input data can be provided in any of the following formats:
+- in the "traditional" array of strings corresponding to the target function's list of input parameters:
+```json
+POST http://localhost:3000/transactions?fly-sync=true&fly-signer=user001&fly-channel=default-channel&fly-chaincode=asset_transfer
+{
+    "headers": {
+        "type": "SendTransaction"
+    },
+    "func": "CreateAsset",
+    "args": ["asset204", "red", "10", "Tom", "123000"]
+}
+```
+- provide a `payloadSchema` property in the input payload `headers`, using [JSON Schema](https://json-schema.org/) to define the list of parameters. The root type must be an `array`, with `prefixItems` to define the sequence of parameters:
+```json
+POST http://localhost:3000/transactions?fly-sync=true&fly-signer=user001&fly-channel=default-channel&fly-chaincode=asset_transfer
+{
+    "headers": {
+        "type": "SendTransaction",
+        "payloadSchema": {
+            "type": "array",
+            "prefixItems": [{
+                "name": "id", "type": "string"
+            }, {
+                "name": "color", "type": "string"
+            }, {
+                "name": "size", "type": "string"
```

Comment:
Is `string` the only valid option for `type` here? For this field since we set the value to `"10"` I would want to set this to an `int` if possible.

Reply:
good suggestion @nguyer